matrix-synapse-riot/terraform/main.tf

```diff
@@ -0,0 +1,250 @@
+# Digital Ocean Tag for all server instances
+resource "digitalocean_tag" "matrix-home-service" {
+  name = "matrix-homeserver"
+}
+
+# Domain name managed by Digital Ocean
```

Comment:
Why commented out?

Reply:
Since we already have the domain already on DO it will fail to create it. So in the README I said it is required to have the domain already on DO DNS.